src/momento-redis-adapter.ts

```diff
@@ -605,6 +617,36 @@ export class MomentoRedisAdapter
     }
   }
 
+  async mset(
+    ...args: [
+      key: RedisKey,
+      value: string | Buffer | number,
+      ...keyValues: (RedisKey | string | Buffer | number)[]
+    ]
+  ): Promise<'OK'> {
+    if (args.length % 2 !== 0) {
+      this.emitError(
+        'mset',
+        "wrong number of arguments for 'mset' command",
+        MomentoErrorCode.INVALID_ARGUMENT_ERROR
+      );
+    }
+    for (let i = 0; i < args.length; i += 2) {
```

Comment:
can't we use setBatch here? maybe you were calling through to `set` in order to get the compression logic for free?

Reply:
Yes, that's right.